src/components/SubmitButton.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
```

Comment:
This button is not meant to be a common component. It contains hard coded copy to the register page and will be even more custom with further animation eventually. Let's keep it where it was and make let's avoid passing custom classnames yet. Each form can have their own submit button.

- remove the classname
- move back under the registration folder
- do not render children.